src/en/community/admin/admin-tooling.md

```diff
@@ -162,12 +202,12 @@ Use the `list` command to list all available commands, and the `help <command na
 | `tpto`                                   | Teleport all targets to the first `<ckey/uid>` in the command.                                       | `tpto <ckey/uid> <ckey/uid> <ckey/uid>`          |
```

Comment:
It may be worth to show optional arguments as ``[ ]``, in this case ``[ckey/uid]``